apic_ml2/neutron/tests/unit/ml2/drivers/cisco/apic/test_cisco_apic_mechanism_driver.py

```diff
@@ -85,13 +85,11 @@
               'topic': 'sometopic', 'agent_type': AGENT_TYPE,
               'configurations': {'opflex_networks': None,
                                  'bridge_mappings': {'physnet1': 'br-eth1'}}}
-AGENT_TYPE_DVS = ofcst.AGENT_TYPE_OPFLEX_OVS
-AGENT_CONF_DVS = {'alive': True, 'binary': 'somebinary',
-                  'topic': 'sometopic', 'agent_type': AGENT_TYPE_DVS,
+AGENT_TYPE_DVS = acst.AGENT_TYPE_DVS
+AGENT_CONF_DVS = {'alive': True, 'binary': 'anotherbinary',
+                  'topic': 'anothertopic', 'agent_type': AGENT_TYPE_DVS,
                   'configurations':
-                      {'hypervisor_type': acst.HYPERVISOR_VCENTER,
-                       'opflex_networks': None,
-                       'bridge_mappings': {'physnet1': 'br-eth1'}}}
+                      {'bridge_mappings': {'physnet1': 'br-eth1'}}}
```

Comment:
Is bridge_mappings relevant for the DVS agent? I don't see this being used by the mechanism driver.